src/tracks/playabletrack/wavetrack/ui/WaveClipUtilities.cpp

```diff
@@ -100,3 +104,17 @@ void PushClipSpeedChangedUndoState(
        XO("Changed Speed to %.01f%%").Format(speedInPercent),
        UndoPush::CONSOLIDATE);
 }
+
+void ShowClipPitchAndSpeedDialog(
```

Comment:
Do not put this here.  I'm going to delete this file in my refactoring branch.  Don't complicate my merges of master into the branch.

Just put in the file where the sole call to it is.

Besides which:  this change is extraneous to what the first commit comment says.

Reply:
I dislike so called "utility" files unless the things put into them really are needed in more than one place.